src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/ListViewUpdater/ListViewUpdater.component.js

```diff
@@ -69,20 +81,27 @@ export const ListViewUpdater = (props: Props) => {
             programId,
             orgUnitId,
             categories,
-            lastIdDeleted,
+            customUpdateTrigger,
         });
         return () => onCancelUpdateList && onCancelUpdateList();
-    }, [
+    }, {
+        forceFirstRunUpdate: forceFirstRunUpdateRef.current,
         filters,
-        sortById,
-        sortByDirection,
-        currentPage,
-        rowsPerPage,
-        programId,
-        orgUnitId,
-        categories,
-        lastIdDeleted,
-    ]);
+        restDependencies: [
+            sortById,
+            sortByDirection,
+            currentPage,
+            rowsPerPage,
+            programId,
+            orgUnitId,
+            categories,
+            customUpdateTrigger,
+            onUpdateList,
+            onCancelUpdateList,
+        ],
+    });
+
+    useEffect(() => () => onCancelUpdateList && onCancelUpdateList(), [onCancelUpdateList]);
```

Comment:
I think I made a similar comment. What is the difference between this and the line 86?